components/supervisor-api/common.proto

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+syntax = "proto3";
+
+package supervisor;
+
+option go_package = "github.com/gitpod-io/gitpod/supervisor/api";
+
+enum TunnelVisiblity {
```

Comment:
nit: should be in `port.proto` ... please avoid kitchen sink packages. Technically it should not make any difference in the generated API/implementation.